test/unit/io/sam_file/sam_file_format_test_template.hpp

```diff
@@ -169,21 +197,7 @@ TYPED_TEST_P(sam_file_read, read_in_all_data)
     typename TestFixture::stream_type istream{this->verbose_reads_input};
     seqan3::sam_file_input fin{istream, this->ref_ids, this->ref_sequences, TypeParam{}};
 
-    this->tag_dicts[0]["NM"_tag] = -7;
-    this->tag_dicts[0]["AS"_tag] = 2;
-    this->tag_dicts[0]["CC"_tag] = 300;
-    this->tag_dicts[0]["cc"_tag] = -300;
-    this->tag_dicts[0]["aa"_tag] = 'c';
-    this->tag_dicts[0]["ff"_tag] = 3.1f;
-    this->tag_dicts[0]["zz"_tag] = "str";
-    this->tag_dicts[1]["bc"_tag] = std::vector<int8_t>{-3};
-    this->tag_dicts[1]["bC"_tag] = std::vector<uint8_t>{3u, 200u};
-    this->tag_dicts[1]["bs"_tag] = std::vector<int16_t>{-3, 200, -300};
-    this->tag_dicts[1]["bS"_tag] = std::vector<uint16_t>{300u, 40u, 500u};
-    this->tag_dicts[1]["bi"_tag] = std::vector<int32_t>{-3, 200, -66000};
-    this->tag_dicts[1]["bI"_tag] = std::vector<uint32_t>{294967296u};
-    this->tag_dicts[1]["bf"_tag] = std::vector<float>{3.5f, 0.1f, 43.8f};
-    this->tag_dicts[1]["bH"_tag] = std::vector<std::byte>{std::byte{0x1A}, std::byte{0xE3}, std::byte{0x01}};
+    this->full_tag_dicts[1]["bH"_tag] = std::vector<std::byte>{std::byte{0x1A}, std::byte{0xE3}, std::byte{0x01}};
```

Comment:
the byte array is ONLY tested here. It should be tested where ever the full tag dictionary is tested but this can be done in another PR